README.md

```diff
@@ -73,15 +75,15 @@ Furthermore, more system dependencies can be installed at build time with the pr
 
 ## Local Development
 
-With `vercel dev` and `vercel-rust`, you can develop your Rust-based lamdas on your own machine.
+With `vercel dev` and `vercel-rust`, you can develop your Rust-based lambdas on your own machine.
 
-During local development with `vercel dev`, ensure `rust` and `cargo` are already installed and available in your `PATH`, since they will not be installed automatically. The recommended way to install `rust` and `cargo` on your machine is with [rustup](https://rustup.rs).
+During local development with `vercel dev`, ensure `rust` and `cargo` are already installed and available in your `PATH`, since they will not be installed automatically. The recommended way to install is with [rustup](https://rustup.rs/).
 
 ## Contributing
 
 Since this project contains both Rust and Node.js code, you need to install the relevant dependencies. If you're only working on the JavaScript side, you only need to install those dependencies (and vice-versa).
 
-```sh
+```
 # install node dependencies
 npm install
 
```

Comment:
Is the rest of the README still correct? i.e. about Cargo workspaces being unsupported?

Reply:
Yes and no, with this new approach you can use workspaces, however you'd need to setup redirects as we have file system based routing. Still need to update the README fully on that regard.

Reply:
Interesting. We should figure out how to make that work nicely.